staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller.go

```diff
@@ -373,3 +445,48 @@ func (c *celAdmissionController) ValidateInitialization() error {
 func (c *celAdmissionController) refreshPolicies() {
 	c.definitions.Store(c.policyController.latestPolicyData())
 }
+
+// validationFailureValue defines the JSON format of a "validation.policy.admission.k8s.io/validation_failure" audit
+// annotation value.
+type validationFailureValue struct {
+	Message           string                      `json:"message"`
+	Policy            string                      `json:"policy"`
+	Binding           string                      `json:"binding"`
```

Comment:
Maybe use `PolicyName` and `BindingName` here since we only assign name not the entire policy/binding?

Reply:
I was trying to save space since I'm already concerned with the data volume produced by these audit annotation values. It should be clear enough? Let me know if you feel strongly about it.

Reply:
Just thought we have Policy/Binding all over and it's easily confused in terms of code maintenance.
It is not a merging blocker though :)